tensorflow_recommenders_addons/dynamic_embedding/python/kernel_tests/horovod_sync_train_test.py

```diff
@@ -27,11 +27,13 @@
 from tensorflow.core.protobuf import config_pb2
 from tensorflow.python.framework import dtypes
 from tensorflow.python.framework import test_util
+from tensorflow.python.framework.errors_impl import NotFoundError
```

Comment:
Considering the PR is almost about the Horovod, we'd better make some UT cases to verify it. Currently, this test file wouldn't be run on regular CI. Could you open it on this PR? (on the BUILD file.)